src/Microsoft.Azure.SignalR/HubHost/NegotiateHandler.cs

```diff
@@ -89,12 +91,14 @@ public async Task<NegotiationResponse> Process(HttpContext context)
                 return null;
             }
 
+            var clientRequestId = _connectionRequestIdProvider.GetRequestId(context.TraceIdentifier);
             var queryString = GetQueryString(
                 request.QueryString.HasValue ? request.QueryString.Value.Substring(1) : null,
-                cultureName,
-                uiCultureName
+                clientRequestId
             );
 
+            _cultureInfoManager.TryAddCultureFeature(clientRequestId, cultureFeature);
```

Comment:
is it possible to check if culture feature is unchanged? If it is unchanged, there is no need to add and apply

Reply:
is it that only blazor app is impacted by the culture?

Reply:
1\ yes, we can check. However if we don't add it, a not found error log will be triggered when trying to apply the culture feature.
2\ added blazor check